### PR TITLE
Replace autoloader to manual loader

### PR DESCRIPTION
**Pull request by koriym**
It doesn't need the autoloader.

### CODE REVIEW

**Comment by koriym** [comment]
@zayoi256 Can you review/confirm again?

**Comment by koriym** [comment]
@izayoi256 ping

**Comment by izayoi256** [comment]
You made a wrong mention, and I wasn't aware of that. I'll give it a try.

File: src/Weaver.php [hunk 1]
```diff
@@ -63,27 +52,27 @@ public function newInstance(string $class, array $args)
     public function weave(string $class) : string
     {
         $aopClass = ($this->aopClassName)($class, $this->bindName);
-        if (! class_exists($aopClass)) {
-            $this->compiler->compile($class, $this->bind);
-            assert(class_exists($aopClass));
+        if (class_exists($aopClass, false)) {
+            return $aopClass;
+        }
+        if ($this->loadClass($class)) {
```
Comment (izayoi256) [inline src/Weaver.php]: I think this line should pass ```$aopClass```, right?

Comment (koriym) [reply src/Weaver.php]: Oh Yes

**Comment by koriym** [comment]
Oh Sorry bad mention.

**Comment by izayoi256** [comment]
@koriym It seems to be okay and better than autoloader.

**Comment by koriym** [comment]
@izayoi256 Less function calls. It's faster.

**Comment by koriym** [comment]
@izayoi256 Released as 2.9.0! https://github.com/ray-di/Ray.Aop/releases/tag/2.9.0

**Pull request merged**